src/main/java/org/gradle/playframework/extensions/PlayDependencies.java

```diff
@@ -0,0 +1,89 @@
+package org.gradle.playframework.extensions;
+
+import org.apache.groovy.util.Maps;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.ExternalModuleDependency;
+import org.gradle.api.provider.Provider;
+
+import javax.inject.Inject;
+
+public class PlayDependencies {
+    private final Project project;
+    private final PlayExtension playExtension;
+
+    @Inject
+    public PlayDependencies(Project project, PlayExtension playExtension) {
+        this.project = project;
+        this.playExtension = playExtension;
+    }
+
+    private static String extractMajorMinorVersion(String version) {
+        return String.join(".", version.split("\\.", 2));
+    }
+
+    private String lazyGetScalaMajorMinorVersion() {
+        return playExtension.getPlatform().getScalaVersion().map(PlayDependencies::extractMajorMinorVersion).get();
+    }
+
+    public Provider<ExternalModuleDependency> evolutions() {
+        return playComponent("play-jdbc-evolutions");
+    }
+
+    public Provider<ExternalModuleDependency> jdbc() {
+        return playComponent("play-jdbc");
+    }
+
+    public Provider<ExternalModuleDependency> javaCore() {
+        return playComponent("play-java");
+    }
+
+    public Provider<ExternalModuleDependency> javaJdbc() {
+        return playComponent("play-java-jdbc");
+    }
+
+    public Provider<ExternalModuleDependency> javaJpa() {
+        return playComponent("play-java-jpa");
+    }
+
+    public Provider<ExternalModuleDependency> filters() {
+        return playComponent("filters-helpers");
+    }
+
+    public Provider<ExternalModuleDependency> cache() {
+        return playComponent("play-cache");
+    }
+
+    public Provider<ExternalModuleDependency> json() {
+        return playComponent("play-json");
+    }
+
+    public Provider<ExternalModuleDependency> ws() {
+        return playComponent("play-ws");
+    }
+
+    public Provider<ExternalModuleDependency> javaWs() {
+        return playComponent("play-java-ws");
+    }
+
+    public Provider<ExternalModuleDependency> specs2() {
+        return playComponent("play-specs2");
+    }
```

Comment:
These methods are pulled from the SBT plugin:

https://github.com/playframework/playframework/blob/master/dev-mode/sbt-plugin/src/main/scala/play/sbt/PlayImport.scala

I think I may need to remove the `json` one because the SBT version I built this list from was old.